build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/ElasticsearchJavaBasePlugin.java

```diff
@@ -196,6 +197,49 @@ private static void configureNativeLibraryPath(Project project) {
         });
     }
 
+    private static void configureEntitlements(Project project) {
+        Configuration agentJarConfig = project.getConfigurations().create("entitlementAgentJar");
+        Project agent = project.findProject(":libs:entitlement:agent");
+        if (agent != null) {
+            agentJarConfig.defaultDependencies(deps -> {
+                deps.add(project.getDependencies().project(Map.of("path", ":libs:entitlement:agent", "configuration", "default")));
+            });
+        }
+        FileCollection agentFiles = agentJarConfig;
+
+        Configuration bridgeJarConfig = project.getConfigurations().create("entitlementBridgeJar");
+        Project bridge = project.findProject(":libs:entitlement:bridge");
+        if (bridge != null) {
+            bridgeJarConfig.defaultDependencies(deps -> {
+                deps.add(project.getDependencies().project(Map.of("path", ":libs:entitlement:bridge", "configuration", "default")));
```

Comment:
```suggestion
                deps.add(project.getDependencies().project(Map.of("path", ":libs:entitlement:bridge")));
```

Relying on the default configuration when declaring a project dependency should be avoided and not necessary here